Makefile

```diff
@@ -1,7 +1,7 @@
 # Placeholder environment variables. These should be present when building
 # and pushing new docker images within the context of github actions.
 # Target version
-VERSION ?= 0.1.5
+VERSION ?= 0.1.6
```

Comment:
I'm not pretty sure this PR is the best place to edit the version. I think the version should be increased when we create a release. Otherwise, two parallel changes would have the same version. WDYT?

Reply:
This one is just a placeholder, it's not used and doesn't have to track the real released version

Reply:
I agree with you. However, there is [0.1.6 release](https://github.com/ondat/metrics-exporter/releases/tag/v0.1.6) published already. This is just aligning the versions.